openchs-server-api/src/main/java/org/openchs/service/OrganisationConfigService.java

```diff
@@ -35,4 +47,33 @@ public OrganisationConfig saveOrganisationConfig(OrganisationConfigRequest reque
         organisationConfigRepository.save(organisationConfig);
         return organisationConfig;
     }
+
+    public LinkedHashMap<String,Object> getOrganisationSettings(Long organisationId) throws JSONException {
+
```

Comment:
Code formatting issues like empty lines, indentation, etc in this file and others. Please format all the changed code in IntelliJ before committing.